AddCode

Ariestellar opened the pull request.

Ariestellar committed:
AddCode

HolyMonkey commented on Assets/Scripts/LevelGenerator.cs:
```diff
@@ -1 +1,144 @@
+        {
+            _numberGridRows = 6;
+            _numberGridCols = 6;
+        }
```
А без этого хардкода никуда?

Ariestellar committed:
Edits №1

Сhanging the operation of the card deck generator through camera control implementation
